### ISSUE TITLE
OEO markdown and csv files

### ISSUE THREAD

**Issue by l-emele** [OpenEnergyPlatform/oeplatform#1123]
Soon, we will be regularly exporting the OEO as markdown file and as csv file, see PR https://github.com/OpenEnergyPlatform/ontology/pull/1319.

These files should be downloadable from the [OEO page](https://openenergy-platform.org/ontology/oeo).

It would be great if the OEP could additionally directly show and render the markdown file.

**Comment by jh-RLI** [comment]
Do you mean someting like this;
![image](https://github.com/OpenEnergyPlatform/oeplatform/assets/38939526/ce1f464a-50db-4fdf-bfbe-246ce1f5da9c)

a new button to download the oeo in markdown format?

**Comment by l-emele** [comment]
Yes, like that: Buttons to download both the markdown and the csv file.

Nice to have would be an additional page that renders the markdown file.

**Comment by stap-m** [comment]
I like the proposal. 
I'd prefer that "owl" stays the most left button and the "md" and "csv" buttons are added to right side of the bar. 
The md and csv are published in the release process in the [existing-terms-and-definitions.zip](https://github.com/OpenEnergyPlatform/ontology/releases/tag/v2.0.1)

**Comment by l-emele** [comment]
While we are just talking about restructuring this page: The files `oeo-full.omn` and `oeo-full.owl` are missing on that page, too.

**Comment by h-spinde** [comment]
To render the markdown file, there are two main options:
One option would be to convert the markdown into html, and then continue to work with the resulting html file.
Alternatively, we could use a tool such as [zero md](https://github.com/zerodevx/zero-md) to render the markdown file directly within the html code of the website.

**Comment by l-emele** [comment]
I thought, the OEP has the capability to render markdown? At least it head, when the tutorials were hosted and displayed on the OEP.

**Comment by jh-RLI** [comment]
We have removed all the associated code as it is no longer used and has been integrated into the Tutorials app only. It's not much work to implement something similar in the Ontology app if someone wants to pick this up i can assist with the implementation.
